### ISSUE TITLE
Looking forward to the latest developments.

### ISSUE THREAD

**Issue by lichengzhang1** [patmorin/extremal-1-planar#1]
Dear Professor Pat Morin,
 The three questions you raised are very attractive to me. I'm currently intrigued by these kinds of questions. Have you made any progress on the three questions you raised? If so, hope to see your outstanding work soon. 

Best wishes,
Licheng Zhang.

**Comment by patmorin** [comment]
Yes, we have a fairly complete answer for $C_3$-free $1$-planar graphs: Such graphs have at most $3(n-2)$ edges and some examples have $3(n-O(1))$ edges.  We have an example of a $C_4$-free $1$-planar graph with $\tfrac 73(n-O(1))$ edges and have worked out most of the details for a matching upper bound. 

We haven't spent much effort yet on $C_5$-free $1$-planar graphs.

**Comment by lichengzhang1** [comment]
It must have been a great piece of work. Thank you very much for letting me know your latest results. Expect to see the official draft soon.

**Comment by lichengzhang2** [comment]
Dear Professor Morin,

I hope you are doing well. A few years ago, I opened this issue to ask about the three extremal questions raised in this repository. I am pleased to let you know that we have recently obtained some related results in this direction.

In our new preprint, Extremal 1-planar graphs without (k)-cliques, joint with Yuanqiu Huang and Fengming Dong, we prove several sharp Turán-type bounds for 1-planar graphs with forbidden cliques. In particular, we improve the upper bound for (K_3)-free 1-planar graphs from (3n-6) to (3n-8), which is tight for all even (n\ge 8). We also determine sharp bounds for (K_4)-free and (K_5)-free 1-planar graphs in the ranges stated in the paper.  

The preprint is available here:
https://arxiv.org/abs/2604.21589

There are still several questions that we have not resolved. For example, the exact extremal value for (K_3)-free 1-planar graphs of odd order remains unclear to us, and it would also be interesting to better understand the structure of extremal or near-extremal examples.

I would be very glad if you find it relevant to the questions in this repository, and I would appreciate any comments or suggestions.

Best wishes,
Licheng Zhang
